src/pip/_internal/req/req_set.py

```diff
@@ -39,8 +46,13 @@ def __repr__(self):
         return ('<%s object; %d requirement(s): %s>'
                 % (self.__class__.__name__, len(reqs), reqs_str))
 
-    def add_requirement(self, install_req, parent_req_name=None,
-                        extras_requested=None):
+    def add_requirement(
+            self,
```

Comment:
Indentation.

Reply:
@pradyunsg Could you provide an example, how it should be indented instead? `self` on the same line as function name, and other params stacked under it?

Reply:
Oh, you mean 
```
    def add_requirement(    
        self,
```
right?

Reply:
Yep.

Reply:
I've fixed indents and imports in this and other PRs.